src/components/enterprise-user-subsidy/UserSubsidy.jsx

```diff
@@ -37,9 +40,6 @@ const UserSubsidy = ({ children }) => {
         return {};
       }
       let hasAccessToPortal = false;
-      if (subscriptionPlan) {
-        hasAccessToPortal = subscriptionLicense?.status === LICENSE_STATUS.ACTIVATED;
-      }
```

Comment:
wait does a license count as an offer? How am I able to access the portal on users with only subscriptions?????
@adamstankiewicz any thoughts on my accidental removal of this?

Reply:
(NVM)
I figured out how this happened.
The logic where this boolean is used only affects the clickability of the Find a course button *inside* the Dashboard page. It does not affect the Header nav's "Find a Course" link which is the only thing I ever consistently click on. Weeeee.

Reply:
Yeah, you're right it doesn't currently apply to the "Find a Course" link in the header, which it probably should conditionally remove the link.

That said, for your question on how you're able to access the learner portal with only subscriptions, look in the `SubscriptionSubsidy` component here: 

https://github.com/edx/frontend-app-learner-portal-enterprise/blob/f199ec7c0a4550f584c3ea0fa9e24b1a1405a576/src/components/enterprise-user-subsidy/SubscriptionSubsidy.jsx#L35-L39

This is what currently controls the redirect back to the Dashboard page if the user doesn't have a valid subscription plan/license and no offers, (e.g. https://enterprise.edx.org/enterprise-demo/search redirects to https://enterprise.edx.org/enterprise-demo if user has no subsidy available to them).

Reply:
Yep. I finally found the SubscriptionSubsidy this morning to have it handy for a new ticket to remove that logic. 😈 
I think I was stuck on the `hasAccessToPortal` variable and thus forgot that it clearly wasn't used for the header and there's different logic going on there. So originally I didn't find the places where we used routing.